src/hooks/useAlternateLinks.ts

```diff
@@ -5,7 +5,7 @@ import { getOrigin } from '../helpers/getOrigin';
 import { getSourceUrl } from '../helpers/getSourceUrl';
 import { useRewrites } from './useRewrites';
 
-interface AlterNateLinks {
+interface AlternateLink {
```

Comment:
Note that you'll need to also update this at the points where it's used further down in this file.